Sources/Views/About.swift

```diff
@@ -29,38 +29,37 @@ public struct Host {
 
 public let aboutExtraStyles = hostImgStyles <> hostBioStyles
 
-public func aboutView(hosts: [Host]) -> [Node] {
-  return [
-    ._gridRow([
-      ._gridColumn(sizes: [.mobile: 12, .desktop: 7], [
-        .div(
-          attributes: [.class([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
-          .fragment(aboutSectionView + openSourceSection)
-        )
-        ]),
+public func aboutView(hosts: [Host]) -> Node {
+  return .gridRow(
+    .gridColumn(
+      sizes: [.mobile: 12, .desktop: 7],
+      .div(
+        attributes: [.class([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
+        aboutSectionView,
+        openSourceSection
```

Comment:
When both of these return `Node` instead they can be inlined, and no need to `.fragment(a + b)`.

Reply:
amazing